### ISSUE TITLE
Don't lose images on template change.

### ISSUE THREAD

**Issue by falcon-git** [h5p/h5p-collage#14]
Should be able to keep images when changing templates and also move images between positions in the template. If moving to a template with fewer images the user should ideally choose the one to loose or get a notice about him loosing the last image or something.

**Comment by icc** [comment]
Agreed, but it will require some work and design to make it work. You must likely wouldn't want the images in the position they are put in. It would be cool with a quick select among the recently uploaded images.

**Comment by falcon-git** [comment]
Yeah, I agree. A quick select among the latest images would be awesome!
